hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/test/MetricsAsserts.java

```diff
@@ -358,6 +358,18 @@ public static void assertGaugeGt(String name, double greater,
         getDoubleGauge(name, rb) > greater);
   }
 
+  /**
+   * Assert that a double gauge metric is greater than or equal to a value.
+   * @param name  of the metric
+   * @param greater value of the metric should be greater than or equal to this
+   * @param rb  the record builder mock used to getMetrics
+   */
+  public static void assertGaugeGte(String name, double greater,
+      MetricsRecordBuilder rb) {
+    Assert.assertTrue("Bad value for metric " + name,
```

Comment:
This assert message could report the greatervalue and the current value.
We should extract:
```
double curValue = getDoubleGauge(name, rb);
```